api/tests/integration/migrations/drop-audit-doc-index.js

```diff
@@ -1,169 +0,0 @@
-var utils = require('./utils');
```

Comment:
Yes. This was the only test using a bit of the framework for testing dbs other than `medic` so I had to do a bunch of setup and teardown just for this test. Because the audit db is no longer used I thought it was ok to remove this test.